src/device-registry/utils/create-event.js

```diff
@@ -168,6 +168,10 @@ const createEvent = {
         return {
           ...item,
           timestamp: item.timestamp.value,
+          gps_device_timestamp:
+            item.gps_device_timestamp && item.gps_device_timestamp.value
```

Comment:
removing the nested time for the gps_device_timestamp when querying mobile data.